shoop/core/models/_contacts.py

```diff
@@ -359,7 +359,7 @@ def get_person_contact(user):
         'last_name': getattr(user, 'last_name', ''),
         'email': getattr(user, 'email', ''),
     }
-    return PersonContact.objects.get_or_create(user=user, defaults=defaults)[0]
+    return PersonContact.objects.filter(is_active=True).get_or_create(user=user, defaults=defaults)[0]
```

Comment:
It shouldn't create a new contact if there already is a contact for the user. The login was supposed to be disabled for those contacts and creating a new contact would allow them to login.